base/shell/explorer/traywnd.cpp

```diff
@@ -522,7 +522,7 @@ class CTrayWindow :
 
     HFONT m_Font;
 
-    HWND m_DesktopWnd;
+    HWND m_ProgMgr;
```

Comment:
Please use the name `m_ProgMan` since this is the standard name under which it is known.

Reply:
OK.